apps/meteor/tests/end-to-end/api/livechat/24-routing.ts

```diff
@@ -91,9 +126,20 @@ import { IS_EE } from '../../../e2e/config/constants';
 			expect(roomInfo.servedBy).to.be.an('object');
 			expect(roomInfo.servedBy?._id).to.not.be.equal(testUser2.user._id);
 		});
+		(IS_EE ? it : it.skip)('should route to contact manager if it is online', async () => {
+			const room = await createLivechatRoom(visitor.token);
+
+			await sleep(5000);
```

Comment:
Is this really necessary?

Reply:
We're doing this in all other routing tests in this describe :thinking: shouldn't we keep doing this, then?

Reply:
yeah, I'm just afraid of removing this and creating a flaky test :cry: 
but I believe we could try removing all sleeps from these tests at some point. I believe we're running all callbacks sync :eyes: